pom.xml

```diff
@@ -139,17 +137,6 @@
             </exclusions>
         </dependency>
 
-        <dependency>
-            <groupId>xml-apis</groupId>
-            <artifactId>xml-apis</artifactId>
-            <version>${xml-apis.version}</version>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.commons</groupId>
-            <artifactId>commons-lang3</artifactId>
-            <version>${commons-lang.version}</version>
-        </dependency>
```

Comment:
Only used in one import, which seems accidental: the rest of the codebase uses Commons Lang 2.x, which is also bundled in Jenkins core and therefore the plugin can be one JAR lighter without the dependency on Commons Lang 3.x.